XPathBenchmarks/Program.cs

```diff
@@ -102,7 +192,7 @@ public object FontoXPath()
             new DynamicContext(
                 new NodeValue<XmlNode>(_source, _domFacade),
                 0,
-                SequenceFactory.CreateEmpty()),
+                SequenceFactory.CreateEmpty(), new Dictionary<string, Func<ISequence>>()),
```

Comment:
missing newline? I'll make a story to integrate a linter. So that code is just always nicely formatted